scripts/startup/nuc_startup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Steps for NUC
+# 1) Start roscore
+# 2) Run nuc-eth-listener
+# 3) Run local pathfinding
+# 4) Run mocks scripts (MOCK_AIS ; MOCK_sensors)
+# No need to start visualizer automatically
+
+NT_ROOT="/home/raye/network-table"
+
+# nuc-eth-listener arguments
+NE_IP="192.168.1.60"
+NE_PORT="5555"
+
+NE_CMD="./build/bin/nuc_eth_listener $NE_IP $NE_PORT"
+
+# Assuming melodica is sourced properly
+
+printf "\nStartup script for NUC\n"
+printf "\n================================\n"
+
+# Roscore
+printf "Roscore\n"
+printf ".....................\n"
+
+if pgrep -f "roscore" > /dev/null;
+then
+    ROS_PID=$(pgrep -f "roscore")
+    printf "Roscore already running with PID: %d\n"$ROS_PID
+else
+    printf "Roscore not started. Starting roscore\n"
+
+    roscore
```

Comment:
I think you need to put the processes in the background (&), or else the subsequent ones don't start